### DISCUSSION TITLE
does etcd release support arm(not arm64)?

### DISCUSSION THREAD

**Discussion by pacoxu** [etcd-io/etcd#15791] [Q&A] [answered]
https://github.com/etcd-io/etcd/blob/7450cd886dd34458e28033588859c662822cb010/scripts/build-release.sh#L27

https://github.com/etcd-io/etcd/blob/217d183e5a2b2b7e826825f8218b8c4f53590a8f/.github/workflows/tests.yaml#L46-L54

I look into this for the failure in https://storage.googleapis.com/kubernetes-jenkins/logs/post-kubernetes-push-image-etcd/1651650057696448512/build-log.txt https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/post-kubernetes-push-image-etcd/1651650057696448512
- some discussions in Slack https://kubernetes.slack.com/archives/CCK68P2Q2/p1682334664191919

**Comment by serathius** [comment, marked as answer]
I think  we build arm, but not release it https://github.com/etcd-io/etcd/blob/main/scripts/release.sh#L261

Official doc about supported architectures https://etcd.io/docs/v3.5/op-guide/supported-platform/
